opensrp-anc/src/main/java/org/smartregister/anc/library/constants/AncAppPropertyConstants.java

```diff
@@ -0,0 +1,8 @@
+package org.smartregister.anc.library.constants;
+
+public class AncAppPropertyConstants {
+    public final static class KEY {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'KEY' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)